api/v1alpha/api/openapi.yaml

```diff
@@ -49220,7 +49434,66 @@ components:
           readOnly: true
           type: string
         eventTypeName:
-          $ref: '#/components/schemas/DefaultEventViewForOrg_eventTypeName'
+          description: Unique identifier of event type.
```

Comment:
I'm amazed how many of those are single reference meaning that oneOf on those were completely pointless.